docker-compose.yml

```diff
@@ -0,0 +1,76 @@
+version: "3.8"
+
+services:
+  localstack:
+    container_name: localstack
+    image: localstack/localstack:1.3.0
+    ports:
+      - "127.0.0.1:5001:5001"
+    volumes:
+      - "./docker/uid2-core/src/init-aws.sh:/etc/localstack/init/ready.d/init-aws-core.sh"
+      - "./docker/uid2-core/src/s3/core:/s3/core"
+      - "./docker/uid2-core/src/kms/seed.yaml:/init/seed.yaml"
+      - "./docker/uid2-optout/src/init-aws.sh:/etc/localstack/init/ready.d/init-aws-optout.sh"
+      - "./docker/uid2-optout/src/s3/optout:/s3/optout"
+    environment:
+      - EDGE_PORT=5001
+      - KMS_PROVIDER=local-kms
+    healthcheck:
+      test: awslocal s3api wait bucket-exists --bucket test-core-bucket
+        && awslocal s3api wait bucket-exists --bucket test-optout-bucket
+      interval: 5s
+      timeout: 5s
+      retries: 3
+
+  core:
+    container_name: core
+    image: ghcr.io/iabtechlab/uid2-core:<CORE_VERSION>
```

Comment:
I think it would be better to set this to uid2-core:latest so that it can still be run locally

Then the regex in the pipeline script should be updated to capture `uid2-core:latest` rather than `<CORE_VERSION>`

Same applies for other versions in this file